contracts/ERC20MultiDelegate.sol

```diff
@@ -0,0 +1,150 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.2;
+
+import {Address} from "@openzeppelin/contracts/utils/Address.sol";
+
+import "@openzeppelin/contracts/access/Ownable.sol";
+import "@openzeppelin/contracts/token/ERC1155/ERC1155.sol";
+import "@openzeppelin/contracts/token/ERC20/extensions/ERC20Votes.sol";
+
+/**
+ * @dev A child contract which will be deployed by the ERC20MultiDelegate utility contract
+ * This is a proxy delegator contract to vote given delegatee on behalf of original delegator
+ */
+contract ERC20ProxyDelegator {
+    constructor(ERC20Votes _token, address _delegatee) {
+        _token.approve(msg.sender, type(uint256).max);
+        _token.delegate(_delegatee);
+    }
+}
+
+/**
+ * @dev A utility contract to let delegators to pick multiple delegatee
+ */
+contract ERC20MultiDelegate is ERC1155, Ownable {
+    using Address for address;
+
+    ERC20Votes token;
```

Comment:
This should be public.

Reply:
updated https://github.com/ensdomains/governance/blob/4e4c7cbc89d82aa9f035a7abcc7a1854432eb423/contracts/ERC20MultiDelegate.sol#L27